my-app/src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,19 @@
+import { NavLink } from "react-router-dom";
+import BurgerMenu from "../../UI-components/BurgerMenu/BurgerMenu";
+import User from "../User/User";
+import style from './Header.module.scss';
+import Search from "../../UI-components/Search/Search";
+
+const Header = () => {
+    return (
+        <div className = {style.header}>
+            <BurgerMenu />
+            <div className = {style.searchUserWrap}>
+                <Search/>
+                <NavLink to = '/user' className = {style.userPoint} onClick={() => console.log('click')}><User userName= {'Artem Malkin'}/></NavLink>
+            </div>
```

Comment:
show this only for authorized user, get information about person from Api, endpoint
/auth/users/me/